AbletonMCP_Remote_Script/handlers/audio.py

```
@@ -0,0 +1,253 @@
+        if getattr(track, "is_frozen", False):
+            return {
+                "success": True,
+                "track_index": track_index,
+                "frozen": True,
+                "track_name": track.name,
+                "message": "Track is already frozen",
+            }
+
+        if not getattr(track, "can_be_frozen", False):
+            raise ValueError(
+                "Track '{0}' cannot be frozen (may be a return/master track or have no devices)".format(track.name))
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)